commit 00fa01698df51c58ae2acf8c7fa4e1fb159f75a9
Author: Jia Tan <jiat0218@gmail.com>
Date:   Tue Jan 9 17:05:01 2024 +0800

    README: Add COPYING.CC-BY-SA-4.0 entry to section 1.1.
    
    The Overall documentation section (1.1) table spacing had to be adjusted
    since the filename was very long.

diff --git a/README b/README
index 9a29c1b2..eaeed4a8 100644
--- a/README
+++ b/README
@@ -67,24 +67,26 @@ XZ Utils
 
 1.1. Overall documentation
 
-    README              This file
-
-    INSTALL.generic     Generic install instructions for those not familiar
-                        with packages using GNU Autotools
-    INSTALL             Installation instructions specific to XZ Utils
-    PACKAGERS           Information to packagers of XZ Utils
-
-    COPYING             XZ Utils copyright and license information
-    COPYING.GPLv2       GNU General Public License version 2
-    COPYING.GPLv3       GNU General Public License version 3
-    COPYING.LGPLv2.1    GNU Lesser General Public License version 2.1
-
-    AUTHORS             The main authors of XZ Utils
-    THANKS              Incomplete list of people who have helped making
-                        this software
-    NEWS                User-visible changes between XZ Utils releases
-    ChangeLog           Detailed list of changes (commit log)
-    TODO                Known bugs and some sort of to-do list
+    README                This file
+
+    INSTALL.generic       Generic install instructions for those not
+                          familiar with packages using GNU Autotools
+    INSTALL               Installation instructions specific to XZ Utils
+    PACKAGERS             Information to packagers of XZ Utils
+
+    COPYING               XZ Utils copyright and license information
+    COPYING.GPLv2         GNU General Public License version 2
+    COPYING.GPLv3         GNU General Public License version 3
+    COPYING.LGPLv2.1      GNU Lesser General Public License version 2.1
+    COPYING.CC-BY-SA-4.0  Creative Commons Attribution-ShareAlike 4.0
+                          International Public License
+
+    AUTHORS               The main authors of XZ Utils
+    THANKS                Incomplete list of people who have helped making
+                          this software
+    NEWS                  User-visible changes between XZ Utils releases
+    ChangeLog             Detailed list of changes (commit log)
+    TODO                  Known bugs and some sort of to-do list
 
     Note that only some of the above files are included in binary
     packages.